perceval/converters/myqlm_converter.py

```diff
@@ -91,11 +91,13 @@ def convert(self, qlmc, use_postselection: bool = True) -> Processor:
 
             if len(instruction_qbit) == 1:
                 ins = None
-                if instruction_name == "H":
-                    ins = Circuit(2, name='H') // BS.H()
-                elif instruction_name == "PH":
-                    phi = instruction[1][0]  # value of the variable parameter in gate
-                    ins = Circuit(2, name='PS') // (1, PS(phi))  # apply phase shift on 2nd mode
+
+                if instruction_name.lower() in catalog.list():
```

Comment:
You can use `if instruction_name.lower() in catalog:` since `Catalog` defines a `__contains__` method